### ISSUE TITLE
Proposal to designate parent lineage of AY.33 and call AY.33 -> AY.33.1

### ISSUE THREAD

**Issue by corneliusroemer** [cov-lineages/pango-designation#219]
New lineage proposal
by @CorneliusRoemer

Description
Sub-lineage of: B.1.617.2 = AY
Earliest sequence: Ghana 2021-05
Most recent sequence: 2021-36
Countries circulating: 30+

The recently designated lineage AY.33 (#215) contains 4 spike mutations:  S:29A, S:250I, S:299I, S:613H. It turns out that there is a fairly wide spread parent lineage that is defined by the first two substitutions S:29A and S:250I but lacks the last two substitutions. It could make sense to designate this parent lineage of AY.33 because the parent without AY.33 is widespread in Denmark, Germany and Sweden, suggesting an epidemiological event. Furthermore, it does have 2 spike mutations which could serve a functional purpose.

In the table below I have listed the number of sequences per country for the parent lineage containing S:29A and S:250I. To motivate why it should be designated in addition to AY.33 I have calculated the number of sequences that are part of this lineage but not of AY.33: 2514 to date, more than AY.33 itself.

Since AY.33 has not been released yet (it seems) it could make sense to designate this lineage as AY.33 and call what used to be AY.33 -> AY.33.1.

Genomes (sequences that belong to AY.33 have been excluded)
[strain_names_clean.txt](https://github.com/cov-lineages/pango-designation/files/7178168/strain_names_clean.txt)
[epi_isl_clean.txt](https://github.com/cov-lineages/pango-designation/files/7178172/epi_isl_clean.txt)

Evidence
![image](https://user-images.githubusercontent.com/25161793/133622697-62baf785-e202-431c-a3b0-f5f31ee46534.png)
https://nextstrain.org/groups/neherlab/ncov/Delta.299I-15Sept?branchLabel=aa&c=gt-S_29,250,299,613&f_clade_membership=21A%20%28Delta%29&gt=S.29A,250I,299T

Country | S:29A, S:250I | AY.33 | S:29A, S:250I without AY.33
-- | -- | -- | --
Denmark | 1011 | 263 | 748
Germany | 763 | 190 | 573
Sweden | 441 | 29 | 412
Turkey | 138 | 0 | 138
Netherlands | 214 | 116 | 98
United States | 171 | 77 | 94
United Kingdom | 251 | 165 | 86
Belgium | 535 | 468 | 67
Switzerland | 107 | 50 | 57
Norway | 68 | 14 | 54
France | 225 | 173 | 52
Finland | 50 | 4 | 46
Bahrain | 18 | 0 | 18
Lithuania | 11 | 0 | 11
Israel | 12 | 2 | 10
Poland | 9 | 0 | 9
Italy | 51 | 45 | 6
Czechia | 5 | 0 | 5
Iceland | 5 | 1 | 4
Ireland | 6 | 2 | 4
Spain | 49 | 45 | 4
Australia | 3 | 0 | 3
Canada | 4 | 1 | 3
Luxembourg | 2 | 0 | 2
Bulgaria | 1 | 0 | 1
Croatia | 1 | 0 | 1
Ghana | 1 | 0 | 1
Japan | 3 | 2 | 1
Jordan | 1 | 0 | 1
Lebanon | 1 | 0 | 1
Romania | 2 | 1 | 1
Slovenia | 1 | 0 | 1
South Korea | 3 | 2 | 1
Sri Lanka | 1 | 0 | 1
Austria | 1 | 1 | 0
Hong Kong | 1 | 1 | 0
Portugal | 2 | 2 | 0
Singapore | 1 | 1 | 0
Sum | 4169 | 1655 | 2514

**Proposed lineage name**
If possible, I would call this lineage AY.33 and rename AY.33 -> AY.33.1

Alternatively, one could withdraw AY.33, call this lineage AY.34 and rename AY.33 -> AY.34.1

**Comment by silcn** [comment]
This proposed parent AY.33 lineage contains a second sublineage that has picked up S:Q613H independently from the proposed AY.33.1; it is shown in orange on the tree above. This lineage is associated with Denmark, and worryingly it appears to be growing rapidly: it has already risen to ~3% prevalence there in the space of little over a month. There are too few sequences in other countries to make any further comparisons. Presumably if this advantage is real it is more likely to be a transmissibility increase than immune escape, given the lack of RBD mutations.

On top of the parent lineage proposed above, this sublineage has ORF1a:K2497R, V2943F and ORF1b:N149D in addition to S:Q613H. Most of the sequences also have N:Q229H.

This lineage could be AY.33.2 or AY.34.2 depending on how the numbering issue is resolved.

**Comment by corneliusroemer** [comment]
@silcn I agree, the presence of a second independent S:613H sublineage that's part of this to be designated parent lineage makes designation of this parent lineage even more worthwhile to reflect the structure in pango lineage names. It's much better to have AY.34, AY.34.1 and AY.34.2 as opposed to AY.33, AY.35, say.

**Comment by FedeGueli** [comment]
AY.34 and AY.35 now designated so probably Ay.33.1/.2 the only way possible now.

**Comment by corneliusroemer** [comment]
@chrisruis @AngieHinrichs @rmcolq 
Have you thought about how to resolve this mess here?
I've been following growth of the three lineages closely and maybe it'd be prudent to simplify things by just designating one catch-all lineage with S:29A and S:250I, since this is the most stable lineage that keeps growing whereas individual sub-lineages are now also going down.

If any of the daughter lineages with S:613H become interesting on their own we can make them sub-lineages. But for now it'd be cleanest to just do the parent one.

Here's up to date evidence:

Defining mutations: S:29A and S:250I (A21647G, A22311T)

Country | Total Variant Sequences | First seq. found at | Last seq. found at |  
-- | -- | -- | -- | --
**Overall** | **6684** | 2021-05 | 2021-37 |
Australia | 3 | 2021-28 | 2021-33 |  
Austria | 79 | 2021-18 | 2021-36 |  
Bahrain | 19 | 2021-29 | 2021-34 |  
Belgium | 713 | 2021-23 | 2021-38 |  
Bulgaria | 1 | 2021-29 | 2021-29 |  
Canada | 90 | 2021-26 | 2021-33 |  
Croatia | 1 | 2021-29 | 2021-29 |  
Czechia | 8 | 2021-31 | 2021-33 |  
Denmark | 1463 | 2021-25 | 2021-37 |  
Estonia | 5 | 2021-32 | 2021-33 |  
Finland | 69 | 2021-26 | 2021-34 |  
France | 520 | 2021-25 | 2021-36 |  
Germany | 1385 | 2021-24 | 2021-38 |  
Ghana | 2 | 2021-05 | 2021-32 |  
Hong Kong | 1 | 2021-34 | 2021-34 |  
Iceland | 5 | 2021-30 | 2021-32 |  
India | 3 | 2021-28 | 2021-29 |  
Ireland | 13 | 2021-30 | 2021-36 |  
Israel | 18 | 2021-18 | 2021-32 |  
Italy | 76 | 2021-27 | 2021-37 |  
Japan | 3 | 2021-23 | 2021-33 |  
Jordan | 1 | 2021-34 | 2021-34 |  
Kuwait | 2 | 2021-33 | 2021-33 |  
Lebanon | 12 | 2021-25 | 2021-28 |  
Lithuania | 26 | 2021-30 | 2021-35 |  
Luxembourg | 2 | 2021-29 | 2021-29 |  
Morocco | 14 | 2021-24 | 2021-34 |  
Netherlands | 301 | 2021-24 | 2021-36 |  
Norway | 111 | 2021-25 | 2021-37 |  
Poland | 13 | 2021-31 | 2021-36 |  
Portugal | 3 | 2021-33 | 2021-35 |  
Romania | 2 | 2021-31 | 2021-31 |  
Singapore | 1 | 2021-33 | 2021-33 |  
Slovakia | 4 | 2021-14 | 2021-35 |  
Slovenia | 1 | 2021-32 | 2021-32 |  
South Korea | 12 | 2021-27 | 2021-35 |  
Spain | 98 | 2021-28 | 2021-38 |  
Sri Lanka | 1 | 2021-24 | 2021-24 |  
Sweden | 618 | 2021-22 | 2021-36 |  
Switzerland | 163 | 2021-23 | 2021-38 |  
Turkey | 205 | 2021-10 | 2021-36 |  
United Kingdom | 325 | 2021-25 | 2021-37 |  
United States | 292 | 2021-23 | 2021-37 |

![image](https://user-images.githubusercontent.com/25161793/134778888-375e33df-a5b7-4752-ad07-c6c1dfdfc645.png)

https://cov-spectrum.ethz.ch/explore/World/AllSamples/AllTimes/variants/json=%7B%22variant%22%3A%7B%22mutations%22%3A[%22S%3A29A%22%2C%22S%3A250I%22]%7D%2C%22matchPercentage%22%3A1%7D

Designating this lineage would also reduce false positives in AY.4 (50% of this lineage is classified falsely as AY.4 at the moment).

Evidence

![image](https://user-images.githubusercontent.com/25161793/134779006-075d12ea-fd4d-4652-bcb9-5dbeb2891666.png)

https://nextstrain.org/groups/neherlab/ncov/Delta.250I?branchLabel=aa&c=gt-S_613,299&dmin=2020-10-19&f_clade_membership=21A%20%28Delta%29&f_region=Europe&m=div

Genomes (~ 5000)
[250I.txt](https://github.com/cov-lineages/pango-designation/files/7230057/250I.txt)

![image](https://user-images.githubusercontent.com/25161793/134779950-3582aee5-b212-49de-bbef-ab37d34808f6.png)

https://nextstrain.org/fetch/genome.ucsc.edu/trash/ct/singleSubtreeAuspice_genome_2b26c_f51ca0.json?branchLabel=Spike%20mutations&label=nuc%20mutations:G4181T,C6402T,C7124T,C8986T,G9053T,C10029T,A11201G,A11332G,C19220T,C27874T,G28916T

**Comment by rmcolq** [comment]
We will be redesignating AY.33 sequences as AY.33.1, and @chrisruis is currently working on selecting the panel of sequences that should go into AY.33 starting at the appropriate point in the tree.

**Comment by chrisruis** [comment]
Thanks @corneliusroemer Based on what you posted above, we've now updated AY.33 in [v1.2.79](https://github.com/cov-lineages/pango-designation/releases/tag/v1.2.79) to start on the branch with A21647G (S:T29A). It looks like S:250I occurs one branch further up than this in the latest UShER tree but an India sequence branches off after S:250I so the introduction(s) into Europe potentially occurred concurrently with S:T29A. So by starting AY.33 on the S:T29A branch, we're starting it with the likely introduction(s) into Europe. We've designated all of this clade as AY.33 for now with no AY.33 sub-lineages but we can add sub-lineages in the future if useful. There's currently 5669 sequences designated AY.33
